examples/authorization_details.json

```diff
@@ -1,12 +1,6 @@
 [
    {
       "type": "openid_credential",
-      "format": "jwt_vc_json",
-      "credential_definition": {
-         "type": [
-            "VerifiableCredential",
-            "UniversityDegreeCredential"
-         ]
-      }
+      "credentials_supported_id": "UniversityDegreeCredential"
```

Comment:
why supported?

`credentials_id` is not enough? see: https://github.com/openid/OpenID4VCI/pull/137#discussion_r1424138842
The meaning that these should be supported is implicit

Reply:
This was consensus in the DCP call

Reply:
argument is that it is pointing to something in the `credentials_supported` object, therefore `credentials_supported_id`

Reply:
I can live with this.